frontend/src/global.scss

```diff
@@ -367,3 +367,27 @@ style files without adding already imported styles. */
         }
     }
 }
+
+// Overlays
+
+.overlay__parent {
```

Comment:
In a general sense, all new styles after the refactor follow a convention with hyphen (i.e. `overlay-parent`), happy to revisit, but I'd be keen on keeping the same convention everywhere.

Reply:
Ah, I was following [BEM](http://getbem.com/), but can go for this simpler convention.

Reply:
Right, might be worth starting to use a convention like that that offers a bit more guidance on how to name styles.